apps/web/package.json

```diff
@@ -29,6 +29,8 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.46.2",
+    "react-json-tree": "^0.18.0",
+    "react-json-view": "^1.21.3",
```

Comment:
is `react-json-view` needed or was this an alternative you tested?

Reply:
Nice catch! It was the latter.